media/css/m24/components/footer-refresh.scss

```diff
@@ -109,21 +109,15 @@ $max-footer-content-width: $content-max;
         display: inline-block;
 
         a {
+            @include bidi(((margin-right, 8px, 0), (margin-left, 0, 8px)));
+            @include image-replaced;
             background-repeat: no-repeat;
+            background-size: 16px 16px;
+            background-position: center center;
             border-bottom: 1px solid rgba(0,0,0,0);
-            direction: ltr;
             display: block;
-            height: 16px;
-            overflow: hidden;
-            text-align: left;
-            text-indent: -9999px;
-            white-space: nowrap;
-            width: 16px;
-            @include bidi(((margin-right, 8px, 0), (margin-left, 0, 8px)));
-
-            @media(min-width: $screen-md) {
-                @include bidi(((margin-right, 16px, 0), (margin-left, 0, 16px)));
-            }
+            height: 24px;
```

Comment:
**Praise:** Larger hit areas 🔥 🔥 🔥

Reply:
Stupid question: Do the hover states have to be separate assets? I see a brief moment when the icon disappears on hover, between hiding white.svg, and loading gray.svg (that had to be duplicated from base protocol here locally just for this color variation).

Upon inspecting the icons, it's `#828282` so for black footer that would be ~50% opacity, now with the alt-black `#161616` it makes ~45% alpha white — do these have to be exact on-brand colors, or could use just reducing opacity from a single white asset?